src/unix/freebsd/cpu.rs

```diff
@@ -140,15 +142,17 @@ pub(crate) struct CpuInner {
     name: String,
     pub(crate) vendor_id: String,
     pub(crate) frequency: u64,
+    pub(crate) arch: String,
 }
 
 impl CpuInner {
-    pub(crate) fn new(name: String, vendor_id: String, frequency: u64) -> Self {
+    pub(crate) fn new(name: String, vendor_id: String, frequency: u64, arch: String) -> Self {
```

Comment:
Why keeping the `String` rather than `CpuArch`?

Reply:
Fixed